src/Bundle.ts

```diff
@@ -306,11 +306,11 @@ function validateOptionsForMultiChunkOutput(
 }
 
 function getIncludedModules(modulesById: Map<string, Module | ExternalModule>): Module[] {
-	return [...modulesById.values()].filter(
-		module =>
+	return Array.from(modulesById.values()).filter(
```

Comment:
Out of curiosity: Does `Array.from` provide any typing benefits over `[... iterator]`? From their behaviour if you do not use the optional second argument of `Array.from`, I would expect them to be more or less the same with the spread operator possibly being slightly more efficient as it needs to do less checking.

Reply:
I think it's more or less the same. dunno about the runtime tho, as the former creates a new array, but all that might be optimized away anyways. 🤷